sources/models/shared-defaults/kubernetes-aws.toml

```diff
@@ -21,3 +21,6 @@ service-checks = ["apiserver", "chronyd", "containerd", "host-containerd", "kube
 
 [metadata.settings.network]
 affected-services = ["containerd", "kubernetes", "host-containerd", "host-containers"]
+
+[settings.autoscaling]
+should-wait = false
```

Comment:
Sorry to be a pain - can you back this specific change out as well? It's tied to the Kubernetes variant models.

Reply:
no prob, i was in doubt if keep it or no, but as you specifically mentioned it before, i leaved it in.